codecov.yml

```diff
@@ -0,0 +1,6 @@
+coverage:
```

Comment:
@maceip I added this coverage config file here. Because the extra ".no_sandbox()" made code coverage checks fail 🤷